manual/source/nxdl-types.rst

```diff
@@ -8,10 +8,10 @@ NXDL: Data Types and Units
 
 .. _nxdl-types:
 		
-Data Types allowed in NXDL specifications
+Field Types allowed in NXDL specifications
```

Comment:
needs more # characters to match length of preceding line

Reply:
Done